packages/bot-api/src/main/MessageHandler.ts

```diff
@@ -37,6 +38,13 @@ abstract class MessageHandler {
     }
   }
 
+  public async sendImage(conversationId: string, image: ImageContent): Promise<void> {
```

Comment:
How about, writing a test for this one ;) ?

Reply:
What should I test here? The parameters are ensured by TypeScript. We should rather write tests in the core for `createImage()` and `send()`. :wink:

Reply:
Wow! I see some logic here. If there is logic, then there is room for tests!
You can check that you don't call the `send` when there is no account or no service.

Reply:
Sure...